app/models/snap_application_attributes.rb

```diff
@@ -152,6 +151,18 @@ def residential_or_homeless
     end
   end
 
+  def phone_attributes
+    return {} if snap_application.phone_number.nil?
```

Comment:
Style: I could be misremembering but I believe we were aiming to avoid explicit returns and trailing conditionals, if we were hewing to the tb style guide. Not a huge deal.

Reply:
I dig it -- updated at b6ac8b2